src/eme.js

```diff
@@ -160,6 +161,10 @@ export const makeNewRequest = (player, requestOptions) => {
           return;
         }
 
+        if (event.messageType === 'license-renewal' && (!player.hasStarted() || player.paused())) {
+          return;
+        }
```

Comment:
This is an attempt to reduce the number of license renewals made when the player is idle (before playback begins) or when the player is paused for too long.

A more ideal solution to this would probably be closing and removing the sessions if the player is abandoned in a paused state and then making a new request when unpaused. Maybe such a behavior should be behind an option where the amount of time to wait (before closing and removing the session) is specified ? But this might introduce some complications.

Reply:
It seems the `paused` check could cause a license to expire mid stream. Has this been tested? Curious if the player can recover the session after playback is resumed.

Reply:
To provide further context to what I'm asking here, I'm curious if the player sits either paused or before starting playback for the duration of a license and it expires, will we request a new license and recover upon resuming playback?

Reply:
Made some changes to the previous logic. I've added an option `limitRenewalsOnPauseTime` (which probably needs a better name) to define the number of seconds to wait in paused state before rejecting renewals and closing the session.

The player will request new license when playback resumes.

Reply:
Maybe it should be something like: `limitRenewalsMaxPauseDuration`